src/main/webapp/stylesheets/teammates.css

```diff
@@ -215,11 +215,35 @@ h2.subcaption {
     display: block;
 }
 
+h2.subcaptionTypeTwo {
+    color: #4379D1;
+    font-size: 15px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    margin: 10px auto;
+    display: block;
+    text-align:center;
+}
+
 .submissionsNumber {
     color: #e46d0b;
     font-size: 20px;
 }
 
+.totalCount {
+    color: #e46d0b;
+    font-size: 20px;
+}
+
+.lastUpdate {
+    position: relative; 
+    bottom: 0px; 
+    left: 600px; 
+    width: 200px; 
```

Comment:
I still think all the styles from `position` to `width` are unnecessary. What do they do?